src/Card/Card.component.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react"
+import styled, { css } from "styled-components"
+import CardBody, { CardBodyProps } from "./CardBody.component"
+import CardFooter, { CardFooterProps } from "./CardFooter.component"
+import CardHeader, { CardHeaderProps } from "./CardHeader.component"
+import CardImage, { ImageProps } from "./CardImage.component"
+import CardFlipper, { CardFlipperProps } from "./CardFlipper.component"
+
+interface CardComposition {

```

Comment:
This should be `ICardComposition` in general best practices for interface naming is prefixed by the letter `I'